frontend/src/metabase-types/api/writeback-form-settings.ts

```diff
@@ -0,0 +1,45 @@
+export type FormType = "inline" | "modal";
+export type FieldType = "text" | "number" | "date" | "category";
+export type InputType =
+  | "text"
+  | "longtext"
```

Comment:
nit: I think `string` and `text` are more common when picking between a short input and a text area